### DISCUSSION TITLE
애드가드에서 LIST-KR 사용시 크롬 깨짐현상

### DISCUSSION THREAD

**Discussion by Escortcat** [List-KR/community#37] [Q&A]
애드가드 평생라이센스를 구매하여 사용중에 있습니다.

현재 사용중인 브라우저는 크롬 109.0.5414.120 빌드이며

이 현상이 발생한지는 약 반년이상 넘었으나 이제야 원인을 파악하여 이곳에 올려봅니다.

![ch2](https://user-images.githubusercontent.com/119315655/215252362-64e284be-a127-4a47-83fd-a3d983e25060.PNG)
![레이아웃1](https://user-images.githubusercontent.com/119315655/215252367-0be6b565-e110-43ab-bde4-2580ef75df18.PNG)
![f12](https://user-images.githubusercontent.com/119315655/215251777-47ef3bc8-918a-461b-8d38-9ac4b8695a38.PNG)

거의 대부분 (90%이상) 크롬에서 깨지기 시작할때는 애드가드에서 LIst-kr 필터가 업데이트 된 시점부터 빈도가 급격히 증가하며

필터를 끄면 증상이 바로 없어집니다.

광고차단을 정상적으로 이용하기 위해서는 애드가드 재시작이 아닌 컴퓨터를 아예 재시작해야 증상이 고쳐집니다.

한국어가 들어있는 페이지에서 주로 발생하며 깨짐현상 뿐만 아니라 컨텐츠가 안보이는 경우도 있습니다.

어떻게 하면 고칠수있을까요

**Comment by piquark6046** [comment]
@Escortcat List-KR 필터를 어떤 방법으로 AdGuard for Windows에 추가하셨나요?
대한민국 금융 보안 프로그램을 설치하셨다면, [List-KR 위키](https://github.com/List-KR/List-KR/wiki/AdGuard-%EB%AC%B8%EC%A0%9C-%EC%A7%84%EB%8B%A8#%EA%B8%88%EC%9C%B5-%EB%B3%B4%EC%95%88-%EC%86%8C%ED%94%84%ED%8A%B8%EC%9B%A8%EC%96%B4%EC%99%80%EC%9D%98-%EC%B6%A9%EB%8F%8C%EB%A1%9C-%EC%9D%B8%ED%95%B4-%EB%B0%9C%EC%83%9D%ED%95%98%EB%8A%94-%EA%B2%BD%EC%9A%B0)에 방법이 있습니다.

**Reply by Escortcat** [reply]
죄송합니다 깃허브 사용을 원체 안하는터라 답변이 늦었습니다.
애드가드 설정 - 광고차단 - 필터 추가 - List-KR 검색 후 구독을 통해 추가하였습니다.
보안프로그램은 구라제거기를 통하여 주기적으로 제거하고있습니다.
애드가드 초기화를 하니 현재까지 문제는 없는듯합니다만 사용시간이 워낙 적어 해결되었는지는 모르겠습니다.

**Reply by piquark6046** [reply]
일단은 문제가 해결되었다고 간주하겠습니다.
나중에 문제가 재발하면 mention해주세요.
